src/scenes/GameScene/GameScene.tsx

```diff
@@ -29,21 +30,47 @@ export function GameScene() {
     });
   };
 
+  useLayoutEffect(() => {
+    // Still need to add a loading state
+    const width = window.innerWidth;
+    const height = window.innerHeight;
+
+    const heightRatio = height / CANVAS_HEIGHT;
+
+    const widthRatio = width / CANVAS_WIDTH;
+
+    if (heightRatio > widthRatio) {
+      // scale based on width
+      setGameScale(widthRatio);
+    } else {
+      // scale based on height
+      setGameScale(heightRatio);
+    }
+    handleStartGame();
+  }, []);
   const randomFragment = <h1 style={{ color: 'green' }}>Daleeeeeee brabo</h1>;
 
   return (
     <>
-      <div>
-        <h1 onClick={() => showPopup(randomFragment)}>Game</h1>
+      <div
+        style={{
+          background: 'white',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}>
+        {/* <h1 onClick={() => showPopup(randomFragment)}>Game</h1> */}
```

Comment:
Pode apagar isso aqui, não?

Reply:
To deixando ai porque a Luiza usa esse pro fragmento do computador, é só pra facilitar na resolução dos conflitos quando fizermos o merge na branch dela.